patches/nginx-1.13.6-daemon-destroy-pool.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/os/unix/ngx_daemon.c b/src/os/unix/ngx_daemon.c
+index ab672110..4bf09e77 100644
+--- a/src/os/unix/ngx_daemon.c
++++ b/src/os/unix/ngx_daemon.c
+@@ -23,6 +23,7 @@ ngx_daemon(ngx_log_t *log)
+         break;
+ 
+     default:
++        ngx_destroy_pool(ngx_cycle->pool);
```

Comment:
Should add a code comment here to make it clear that we just make it "ASAN clean".